scripts/create_releases_pr.sh

```diff
@@ -0,0 +1,100 @@
+#!/bin/bash
+
+set -e
+
+export DOCS_SITE_BASE_DIR="."
+export GITHUB_ORG_NAME=gruntwork-io
+
+export RELEASES_PLUGIN_BEFORE_DATE=
+export BRANCH_NAME=
+export CONFIGURE_GIT=${CONFIGURE_GIT:=1}
+
+function set_release_date() {
+  unameOut="$(uname -s)"
+
+  # Handling running this on linux or mac
+  if [ ${unameOut} = "Linux" ]; then
+    RELEASES_PLUGIN_BEFORE_DATE=$(date -d "$(date +"%Y-%m-01") - 1 day" +"%F")
```

Comment:
Is there any issue with having nested double quotes like this?

Reply:
It works the same with single quotes in there (i have to use gdate on my mac to get the gnu date syntax)

![image](https://github.com/gruntwork-io/docs/assets/25206261/1e9e256d-19df-4d7c-a2a0-27abe60ba82e)


![image](https://github.com/gruntwork-io/docs/assets/25206261/a99e55f2-ed07-4c6c-b8ef-311df814cd9c)